### PR TITLE
Add driver code for roko7 project

### PR DESCRIPTION
**Pull request by dustbin2410**
dmmT

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# zuzu_roko7
+Fok from zuzu/roko
+Add Project description:
```
Comment (zhouyu2410) [inline README.md:4]: dm desc ccc, cuuttt

File: README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# zuzu_roko7
```
Comment (zhouyu2410) [inline README.md:1]: dmm,. zhouyu NOT zuzu

File: README.md (hunk 1 above)
Comment (dustbin2410) [reply README.md:4]: ccc là ccc gì, dmm zuzu

Comment (zhouyu2410) [reply README.md:4]: ok I accept your apology bindog. dmm dostbin

**Pull request merged**